src/app/shapes/Text.ts

```diff
@@ -0,0 +1,32 @@
+import * as Konva from "konva";
+import { Shape } from "./Shape";
+
+export class Text extends Shape {
+
```

Comment:
I think Text is already a class, we might have to call this TextShape or something